content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -142,6 +142,45 @@ To create a new Secret, perform the following steps:
 1. Place that value in the `secret` field of the `EncryptionConfiguration` struct.
 1. Set the `--encryption-provider-config` flag on the `kube-apiserver` to point to
    the location of the config file.
+
+   You will need to mount the new encryption config file to the `kube-apiserver` static pod. Here is an example on how to do that:
```

Comment:
```suggestion
   Mount the new encryption config file to the `kube-apiserver` static pod.
```

Reply:
The main takeaway is to remove the word `will` based on k/website's [style-guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense) requiring present tense. The other parts of the suggestion are non-blocking.